src/screens/ProfileScreen/OneOnOneList/OneOnOneList.js

```diff
@@ -0,0 +1,43 @@
+import React, { useEffect } from 'react';
+import { FlatList, View } from 'react-native';
+import { useDispatch, useSelector } from 'react-redux';
+
+import { getOneOnOnes } from 'actions/oneOnOneActions';
+import strings from 'locale';
+
+import { Text } from 'components';
+import OneOnOneItem from './OneOnOneItem';
+
+import styles from './OneOnOneList.styles';
+
+const OneOnOneList = () => {
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(getOneOnOnes());
+  }, [dispatch]);
```

Comment:
What about a hook useOneOnOnes?

Reply:
hmm it's not gonna be reused, I rather leave it as is